reports/summary.md

```diff
@@ -140,7 +140,6 @@ Assets that appear in app-data but do not have references within sheets
 | ios/icon.png | 128.1 |
 | ios/splash.png | 147.4 |
 | map_data/forecast_data.json | 50.2 |
```

Comment:
@jfmcquade Can I remove this unused asset? I double checked [comp_map_layer_data](https://docs.google.com/spreadsheets/d/1KgLTrFhqZdGhcSk6Zt0KQoy2mYf7NCqhCJz_zS0KEYw/edit?gid=1695224840#gid=1695224840) and it doesn't seem to be used